connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -339,8 +340,15 @@ private PluginScanResult scanPluginPath(
             Class<T> klass,
             ClassLoader loader
     ) throws InstantiationException, IllegalAccessException {
-        Set<Class<? extends T>> plugins = reflections.getSubTypesOf(klass);
-
+        Set<Class<? extends T>> plugins;
+        try {
+             plugins = reflections.getSubTypesOf(klass);
+        } catch (ReflectionsException e) {
+            log.debug("Reflections scanner could not find any classes for URLs {}",
```

Comment:
Could it print the origin exception also? ```log.debug("xx", e)```

Reply:
Good idea. Done.